joy2twist/launch/gamepad_controller.launch.py

```diff
@@ -0,0 +1,45 @@
+import os
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, ExecuteProcess, IncludeLaunchDescription
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import Node
+
+
+def generate_launch_description():
+
+    joy2twist_cfg_path = os.path.join(get_package_share_directory(
+        'joy2twist'), 'config', 'joy2twist.yaml')
+
+    joy2twist_params_file_argument = DeclareLaunchArgument(
+        'joy2twist_params_file',
+        default_value=joy2twist_cfg_path,
+        description='ROS2 parameters file to use with joy2twist node')
+
+    joy2twist_launch = create_include_launch(
+        package='joy2twist',
+        rel_launch_path='/launch/joy2twist.launch.py',
+        arguments={'joy2twist_params_file': LaunchConfiguration('joy2twist_params_file')})
+
+    joy_linux_node = Node(
+        package='joy_linux',
+        executable='joy_linux_node',
+        output={
+            'stdout': 'screen',
+            'stderr': 'screen'
+        },
+        emulate_tty='true')
+
+    actions = [joy2twist_params_file_argument,
+               joy2twist_launch, joy_linux_node]
+
+    return LaunchDescription(actions)
+
+
+def create_include_launch(package: str, rel_launch_path: str, arguments: dict):
```

Comment:
I'm not sure if I caught it earlier. Do we really want to use type hinting?

Reply:
To be honest I guess that this was done by formatter. Nevertheless, this improves the readability of a code in my opinion. We need to discuss it in a larger group and find the pattern.

Reply:
I also think that it improves readability and I vote for using it.

Reply:
Stays! :grinning: